client/src/hooks/useIncTypeOptions.js

```diff
@@ -17,42 +17,43 @@ export function useIncTypeOptions() {
 
   const [incTypesOptions, setIncTypesOptions] = useState();
 
-  const covidKeywords = ["covid19","covid","coronavirus"];
-
-  const covidException = (desc, keywords) => {
+  const checkCovidSpecialCase = (desc, keywords) => {
     const lowercasedDescription = desc.toLowerCase();
     return (
-      keywords.find((keyword) =>
-          lowercasedDescription.includes(keyword.toLowerCase()
-          )
+      keywords.find((keyword) => lowercasedDescription.includes(keyword.toLowerCase())
       ) || null
     );
   };
 
   const updateOptionsFromDescription = useCallback(
     async (description, options) => {
-       // specialcase to be handled for Covid keywords
+      // specialcase to be handled for Covid keywords
+      const covidKeywords = ["covid", "coronavirus"];
 
       if (options) {
-        if(covidException(description,covidKeywords)!= null){
-        
-          setIncTypesOptions(Object.values(options));
-          setIncidentTypePriAutocomplete({label: "COVID-19 Confirmed", 
-                                          value: "COVID-19 Confirmed", 
-                                          confidence: "1"});
-         
-          }
-        else{
-        const { updatedIncTypes, topIncType } = await getMultiPrediction(
-          description,
-          options
+        const covidOptionExist = options.filter((type) =>
+          type["label"].toLowerCase().includes("covid-19 confirmed")
         );
-        
-        setIncTypesOptions(Object.values(updatedIncTypes));
-        setIncidentTypePriAutocomplete(topIncType);
+
+        if (
+          checkCovidSpecialCase(description, covidKeywords) != null &&
+          covidOptionExist != null
```

Comment:
`options.filter` will return an empty array if it doesn't find anything so this won't run. It'll get fixed by the changing the call from `filter` to `find`. Change this to `!covidOptionIfExists` to catch both `null` and `undefined`.